new-docs/content/any/_index.md

```diff
@@ -3,26 +3,10 @@
 
 ## What is Oso?
 
-Oso is an batteries-included, open-source library for building authorization
-into your application.
+Oso is a batteries-included library for building authorization in your application. 
 
-Oso helps you form a mental model for your
-authorization needs by providing a set of frameworks and templates built on
-top of a
-declarative policy language called Polar. The Oso library provides a
-simple API for querying Polar policies, which centralize authorization
-logic, and middleware to help apply authorization in the right places. Oso
-also provides a debugger and REPL for Polar, to help developers learn the
-language and test their policies.
+Oso gives you a mental model and an authorization system – a set of APIs built on top of a declarative policy language called Polar, plus a debugger and REPL – to define who can do what in your application. You can express common concepts from “users can see their own data” and role-based access control, to others like multi-tenancy, organizations and teams, hierarchies and relationships.
 
+Oso lets you offload the thinking of how to design authorization and build features fast, while keeping the flexibility to extend and customize as you see fit.
 
-Oso lets you offload the thinking of how to design authorization and build
-features fast, while keeping the flexibility to extend and customize as you
-see fit.
-
-Developers can typically write a working policy in <5 minutes, add Oso to an
-app in <30 minutes, and use Oso to solve real authorization problems in a day
-or two. To get started, you add the library to your application, create a new
-Oso instance and load an Oso policy. You can mix and match any of Oso's
-authorization templates to implement features like roles while also writing
-custom policies that suit your application.
+Developers can typically write a working Oso policy in <5 minutes, add Oso to an app in <30 minutes, and use Oso to solve real authorization problems in 1 day. To get started, you add the library to your application, create a new Oso instance and load an Oso policy. You can mix and match any of Oso's authorization APIs to implement features like roles with custom policies that you write to suit your application.
```

Comment:
1 day sounds like a while. how about "within a few hours"?

Reply:
works for me, thanks gabe